azure-template/sdk_packaging.toml

```diff
@@ -0,0 +1,7 @@
+[packaging]
+auto_update = false
```

Comment:
We might discuss if it's valuable to use more automatic update, instead of less. It drives consistency, and I'd rather have a little more extension that to turn it off by default. TBD

Reply:
I set it true. If we need to change it later, alright.